packages/react/src/components/Popover/Popover.stories.tsx

```diff
@@ -50,37 +50,47 @@ Preview.args = {
 };
 
 export const Variants: StoryFn<typeof Popover> = () => {
-  const [buttonRef, setButtonRef] = useState<HTMLButtonElement | null>(null);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+  const [open, setOpen] = useState(false);

```

Comment:
This was default open before, its now closed, was this intended?

Reply:
We could add a use effect that sets it to `true`, but it needs a rerender for the `Popover` to position itself

Reply:
Hehe, yeah, we did that before, and just added a comment which said this was to show the popover open on story load.